ethereumj-core/src/test/java/org/ethereum/core/PendingStateTest.java

```diff
@@ -119,7 +119,7 @@ public void changeForkTest() {
         pendingState.processBest(b1);
         assertEquals(pendingState.getPendingTransactions().size(), 0);
 
-        Block b1_ = blockchain.createNewBlock(blockchain.getBestBlock(), Arrays.<Transaction>asList(), Collections.EMPTY_LIST);
+        Block b1_ = blockchain.createNewBlock(blockchain.getBestBlock(), Collections.EMPTY_LIST, Collections.EMPTY_LIST);
```

Comment:
Actually I’ve tried to do it all over the code but I get a compile time error. There is a significant number of unchecked casts all over the code (just compile with `-Xlint:unchecked`). If we fix them, we might be able to use this static method. In this case I’ve reverted the the `emptyList()` that I’ve introduced in a previous commit.